bfsdk/include/bfsupport.h

```diff
@@ -116,16 +132,30 @@ struct section_info_t {
  *      dummy member to avoid an empty struct on platforms not needing platform info
  */
 struct platform_info_t {
+
+    /// Signature
```

Comment:
Same thing will all of these, no comments in the struct, they should be in the above comment block using "var"